src/benchmark.zig

```diff
@@ -1,3 +1,24 @@
+// This benchmark uses a single client to 1) create `account_count`
```

Comment:
Let's maybe start with a big picture, how this file fits with the rest of the codebase (is this a micro benchmark? a macro benchmark? a whole bench or a part), and only then go into the detail of what actually this benchmarks. 

Something like 

> benchmark.zig is a program which connects to a tigerbeetle cluster and runs a predefined load against it, measuring observed latencies and throughput. See benchmark.{sh,bat} for the full benchmarking setup.

Reply:
Good idea! Done.